.gitignore

```diff
@@ -1,3 +1,7 @@
 /node_modules/
 /TODO
 /NO_GIT/
+/lib/libWebRTC-LATEST-Universal-Release.a
```

Comment:
I don't understand why `libWebRTC-LATEST-Universal-Release.a` is removed from the repository.